<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Captura Alumnos</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Materia:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parcial:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Calificaci√≥n:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNombre">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMaterias">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbParciales">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="dsbCalificacion">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAgegar">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Agregar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSalir">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblAlumnnos">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>581</width>
     <height>361</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
